core/shared/src/main/scala/zio/internal/metrics/package.scala

```diff
@@ -0,0 +1,10 @@
+package zio.internal
+
+package object metrics {
+
+  private[zio] lazy val metricState: ConcurrentState =
```

Comment:
Ah this can go in platform too, perhaps.

Reply:
Yes. There is a little thinking to be done here about how the ability to potentially change the platform interacts with this.